misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/TestCmd.java

```diff
@@ -75,13 +75,13 @@
     @CommandLine.Option(names = {"--list-groups", "-lg"}, description = "list the groups available in the tests")
     private boolean listGroups;
 
-    @CommandLine.Option(names = "--groups", description = "test groups to be executed")
+    @CommandLine.Option(names = "--groups", split = ",", description = "test groups to be executed")
```

Comment:
Is it possible to write tests for these?

Reply:
would make sense to have them as part of the tests for the Test command, WDYT? AFAIK, we are yet to introduce such tests.

Reply:
I think it is better if we can add integration tests for `test` command similar to what we have done for `run`, `build`, etc commands in https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/PackagingInitTestCase.java.

Reply:
Yes, the plan is to add tests for the individual commands.

Reply:
@MaryamZi we already have integration tests for the test command

Reply:
@MaryamZi and @Shan1024 we have integration tests for all the individual commands including the test command in https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/ballerina-tools-integration-test/src/test/java/org/ballerinalang/test/packaging/TestExecutionTestCase.java

Reply:
@NatashaWso2, thank you, missed this! Will add tests for the test command with multiple groups, to capture the separator.

Reply:
Added with e26f1fb